k8s/namespaces/nfdiv/nfdiv-cron-applicant1-apply-conditional-order/nfdiv-cron-applicant1-apply-conditional-order.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: nfdiv-cron-applicant1-apply-conditional-order
+spec:
+  releaseName: nfdiv-cron-applicant1-apply-conditional-order
+  chart:
+    git: git@github.com:hmcts/nfdiv-cron
+    ref: 0.0.8
+    path: nfdiv-cron
+  values:
+    job:
+      image: hmctspublic.azurecr.io/nfdiv/case-api:prod-5be3063-20210920170842 #{"$imagepolicy": "flux-system:nfdiv-case-api"}
+      keyVaults:
+        nfdiv:
+          secrets:
+            - name: AppInsightsInstrumentationKey
+              alias: APP_INSIGHTS_KEY
+            - name: uk-gov-notify-api-key
+              alias: UK_GOV_NOTIFY_API_KEY
+            - name: s2s-case-api-secret
+              alias: S2S_SECRET
+            - name: idam-secret
+              alias: IDAM_CLIENT_SECRET
+            - name: idam-systemupdate-username
+              alias: IDAM_SYSTEM_UPDATE_USERNAME
+            - name: idam-systemupdate-password
+              alias: IDAM_SYSTEM_UPDATE_PASSWORD
+      environment:
+        TASK_NAME: SystemNotifyApplicant1ApplyForConditionalOrder
+      schedule: 0/10 * * * *
```

Comment:
I think this schedule format was changed by platops I think it was `*/10 * * * *` @linusnorton  ?

Reply:
ok will change, just followed the approach taken with the other jobs.

Reply:
That format doesn't appear to be valid yaml @nitinprabhu

Reply:
I got confused I think